addons/event/tests/test_event_internals.py

```diff
@@ -452,12 +452,13 @@ def test_event_seats(self):
 
         # create registration in order to check the seats computation
         self.assertTrue(event.auto_confirm)
-        for x in range(5):
-            reg = self.env['event.registration'].create({
-                'event_id': event.id,
-                'name': 'reg_open',
-            })
-            self.assertEqual(reg.state, 'open')
+        reg_open = self.env['event.registration'].create([{
+            'event_id': event.id,
+            'name': 'reg_open',
+        } for _ in range(5)])
+
+        reg_open = reg_open[0]
```

Comment:
Strange to re-us the same naming. Either use reg_open[0] herebelow, either maybe call it differently (reg_open_test) ?